src/pretix/control/views/checkin.py

```diff
@@ -69,6 +70,34 @@ def get_context_data(self, **kwargs):
                            or "subevent" in self.request.GET)
         return ctx
 
+    def post(self, request, *args, **kwargs):
+        for c in request.POST.getlist('checkin'):
+            created = False
+            op = OrderPosition.objects.select_related('item', 'variation', 'order', 'addon_to').get(
+                order__event=self.request.event,
+                order__code=c
+            )
```

Comment:
See above, replace by ``pk=c``. We could also optimize this by restructuring it the following way:

```
 positions = OrderPosition.objects.select_related('item', 'variation', 'order', 'addon_to').filter(
    order__event=self.request.event,
    pk__in=request.POST.getlist('checkin')
)
for op in positions:
```
This only requires one SQL query instead of ``N``.